### PR TITLE
Add documentation for NodeSwap tech preview feature

### PR DESCRIPTION
**Pull request by ehashman**
Ref https://issues.redhat.com/browse/OCPNODE-791

/cc @sjstout @mburke5678

### CODE REVIEW

**Comment by mburke5678** [comment]
@ehashman Good start here! 

A few questions:
* If the user need to [Add swap to nodes](https://github.com/openshift/enhancements/blob/master/enhancements/kubelet/node-swap.md#add-swap-to-nodes), should we include a step? Does this cover the [_Nodes must already have swap memory provisioned._ prerequisite](https://github.com/openshift/openshift-docs/pull/41755/files#diff-970bc4c39872aa35f69b91ae8bc0644a18bad80e0b4beb20672a7c221c06f831R20)?

File: modules/nodes-nodes-swap-memory.adoc [hunk 1]
```diff
@@ -0,0 +1,50 @@
+:_content-type: PROCEDURE
+[id="nodes-nodes-swap-memory_{context}"]
+
+= Managing swap memory on nodes
+
+:FeatureName: Configuring swap memory usage on nodes
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+In {product-title}, you can configure nodes to use swap memory. By default, nodes will not start in the presence of swap memory.
```
Comment (mburke5678) [inline modules/nodes-nodes-swap-memory.adoc:9]: | By default, nodes will not start in the presence of swap memory.
What does this mean?

Comment (mburke5678) [reply modules/nodes-nodes-swap-memory.adoc:9]: Do we need to explain what swap memory is?

File: modules/nodes-nodes-swap-memory.adoc [hunk 2]
```diff
@@ -0,0 +1,50 @@
+:_content-type: PROCEDURE
+[id="nodes-nodes-swap-memory_{context}"]
+
+= Managing swap memory on nodes
+
+:FeatureName: Configuring swap memory usage on nodes
+include::modules/technology-preview.adoc[leveloffset=+1]
```
Comment (mburke5678) [inline modules/nodes-nodes-swap-memory.adoc:7]: Apparently, this file has been moved. 
`include::snippets/technology-preview.adoc[]`

**Comment by ehashman** [comment]
>     * If the user need to [Add swap to nodes](https://github.com/openshift/enhancements/blob/master/enhancements/kubelet/node-swap.md#add-swap-to-nodes), should we include a step? How much of the `systemd.units.contents` parms do we need to define? Does this cover the [_Nodes must already have swap memory provisioned._ prerequisite](https://github.com/openshift/openshift-docs/pull/41755/files#diff-970bc4c39872aa35f69b91ae8bc0644a18bad80e0b4beb20672a7c221c06f831R20)?

I don't think we need to include instructions on how to add swap memory to nodes. The linked instructions from the enhancement are mostly for convenience in testing/development for a cloud environment. The performance would not be sufficient for most production use cases.

Would I be able to link to https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/managing_storage_devices/getting-started-with-swap_managing-storage-devices ?

>     * I assume that a typical user of this feature knows how to [enable swap accounting on the nodes](https://github.com/openshift/openshift-docs/pull/41755/files#diff-970bc4c39872aa35f69b91ae8bc0644a18bad80e0b4beb20672a7c221c06f831R21)?

This is a single kernel parameter, e.g. https://access.redhat.com/solutions/453813

>     * For limited swap, should/can we provide some info on what the limitation is?

It depends on the version of cgroups that one is running with. More details available in https://kubernetes.io/docs/concepts/architecture/nodes/#swap-memory

I could copy that to the OpenShift docs.

**Comment by mburke5678** [comment]
`Would I be able to link to https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/managing_storage_devices/getting-started-with-swap_managing-storage-devices ?`
Yes

```
It depends on the version of cgroups that one is running with. More details available in https://kubernetes.io/docs/concepts/architecture/nodes/#swap-memory
I could copy that to the OpenShift docs.
```
Sound good!

**Pull request merged**